clients/python/run-tests.sh

```diff
@@ -0,0 +1,3 @@
+build/sbt pythonClient/generate
```

Comment:
dont you need `!/bin/sh` thing? or should dev run it as `sh run-tests.sh`. this should be written in the readme.